client/payment-details/timeline/map-events.js

```diff
@@ -253,10 +262,12 @@ export const composeFeeString = ( event ) => {
 
 	const {
 		percentage,
-		fixed,
 		fixed_currency: fixedCurrency,
 		history,
 	} = event.fee_rates;
+	// The fixed part of the fee is a float in cents, so we need to prevent rounding issues.
+	const fixed = Math.round( event.fee_rates.fixed );
```

Comment:
Also, the fixed part is supposed to be an integer. Do we know the trail where floats are getting in?

Reply:
Yes I tried to answer in previous comment, but in a nutshell it comes from the server where fixed_rate is defined as float